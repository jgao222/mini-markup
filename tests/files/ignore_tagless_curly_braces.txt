<tag> { some text { no tag? } }