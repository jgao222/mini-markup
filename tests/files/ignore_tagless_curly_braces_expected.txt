<tag> some text { no tag? } </tag>